generator/generator_tests/transit_tools.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "testing/testing.hpp"
+
+#include <vector>
+
+namespace routing
+{
+namespace transit
+{
+template<typename Obj>
+void TestForEquivalence(std::vector<Obj> const & objects, std::vector<Obj> const & expected)
+{
+  for (size_t i = 0; i < objects.size(); ++i)
```

Comment:
s/objects/actual/
How about to check that ```actual.size() == expected.size()```? Is it a case? If so, please, add a test that reveals the problem here.

Reply:
> s/objects/actual/

Done

> How about to check that actual.size() == expected.size()? Is it a case? If so, please, add a test that reveals the problem here.

It's exactly the case. Added.